src/html_support_files/odoc.css

```diff
@@ -1193,21 +1210,53 @@ td.def-doc *:first-child {
 
 /* Mobile adjustements. */
 
+@media only screen and (max-width: 210ex) {
```

Comment:
Maybe this could be 160ex because this is the size of the body in line 299. Maybe even make it into a variable ?